addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -362,6 +362,8 @@
                                     <group string="Advanced" groups="base.group_no_one">
                                         <field name="mail_server_available" invisible="1"/>
                                         <field name="name" required="False" string="Name" attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}"/>
+                                        <field name="bypass_blacklist"
+                                               attrs="{'readonly': [('state', 'not in', ('draft'))]}"/>
```

Comment:
does this work ?
(it will be interpreted as `['d', 'r', 'a', 'f', 't']` because `('draft')` is not a tuple but a string)
```suggestion
                                               attrs="{'readonly': [('state', 'not in', ('draft', ))]}"/>
```

Reply:
Indeed, thanks, I have changed it. (but strangely, it works also without the comma...)